### PR TITLE
Updated package.json for security vulnerabilities

### PR DESCRIPTION
**Pull request by sarvex**

### CODE REVIEW

**Comment by ascorbic** [comment]
Hi. Can you add a description and explain the rationale for each of the things you want to add here? We already use Renovate, so there's no reason to add Dependabot.

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,12 @@
+sonar.projectKey=sarvex_unpic-img
+sonar.organization=sarvex
```
Comment (ascorbic) [inline sonar-project.properties:2]: Is this your organization?

Comment (sarvex) [reply sonar-project.properties:2]: @ascorbic yes they are from open source sonar cloud configuration. 
Need to exclude these from the PR

**Comment by ascorbic** [comment]
Closing because there's not a good rationale for why it's needed.

**Pull request closed without merging**